mineinabyss-components/src/main/kotlin/com/mineinabyss/components/armorstandlock/LockArmorStand.kt

```diff
@@ -0,0 +1,14 @@
+package com.mineinabyss.components.armorstandlock
+
+import com.mineinabyss.idofront.serialization.UUIDSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.util.*
+
+@Serializable
+@SerialName("mineinabyss:lockable")
+class LockArmorStand(
```

Comment:
Make this a data class, always default to it for components unless complications arise because of it.